docs/lua/functions-3.md

```
@@ -139,6 +139,27 @@ Sets the current object's position to random floats between 0.0 and 1.0
+`local booleanValue = draw_distance_scalar_is_infinite()`
+
+### Parameters
+- None
+
+### Returns
+- `boolean`
+
+### C Prototype
+`bool draw_distance_scalar_is_infinite(void);`
+
+[:arrow_up_small:](#)
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)